poll/public/html/poll.html

```diff
@@ -10,8 +10,10 @@ <h3 class="poll-header">{{display_name}}</h3>
       <ul class="poll-answers">
       {% for key, value in answers %}
           <li class="poll-answer">
-              <input type="radio" name="choice" id="{{url_name}}-answer-{{key}}" value="{{key}}" {% if choice == key %}checked{% endif %}/>
-              {% if value.img %}
+              <div class="poll-input-container">
+                  <input type="radio" name="choice" id="{{url_name}}-answer-{{key}}" value="{{key}}" {% if choice == key %}checked{% endif %}/>
+              </div>
+              {% if any_img %}
               <div class="poll-image">
```

Comment:
@Kelketek Nit: Since you touched the `if` above you could take this opportunity to properly indent L17 through L23.